src/languages/de-DE.json

```
@@ -0,0 +1,278 @@
+	"global_logging_unban_auto": "`[{timestamp}]` {emoji} **{target_tag}** `({user_id})`wurde für {action} von **{moderator_tag}** `({moderator_id})` Nachdem die Strafe abgelaufen ist `[#{infractionId}]`",
+	"global_logging_mod_temp": "`[{timestamp}]` {emoji} **{target_tag}** `({user_id})`wurde für {action} von **{moderator_tag}** `({moderator_id})` bis **{until}** für **{reason}** `[#{infractionId}]`",
+	"global_no_reason": "Keine Begründung gegeben",
+	"global_loading": "{emote_loading} Lädt...",
+	"global_not_in_voice": "{emote_warn} Benutzer ist mit keinem Voice-Channel verbunden!",
+	"global_unknown_error": "{emote_warn} Ein Unbekannter Fehler ist aufgetreteb, Bitte versuche es später wieder wenn der Fehler immer noch ist, kontaktiere bitte einen Entwickler",
+
+	"global_cannot_action_self": "{emote_warn} Du kannst diese Aktion nicht an dir Selbst durchführen!",
+	"global_cannot_action_owner": "{emote_warn} Du kannst diese Aktion nicht am Server-Besitzer durchführen!",
+	"global_cannot_action_role_equal": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, da du keine Höhere Rolle als der User hast!",
+	"global_cannot_action_role_equal_bot": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, da meine Rolle niedriger ist als die Rolle vom User!",
+	"global_cannot_action_role_higher": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, Da er eine höhere Rolle als du hast!",
+	"global_cannot_action_role_higher_bot": "{emote_warn} Du kannst diese Aktion nicht an **{user_tag}** durchführen, da er eine Höhere Rolle als ich hat!",
+	"global_cannot_action_bot_self": "{emote_warn} You cannot perform this action on the bot!",
+
+	"premium_message": "Das ist ein Premium Befehl, Werde Patreon und bekomme Zugriff auf die Premium Befhele <https://www.patreon.com/bulbbot>",
+
+	"event_message_args_unexpected": "{emote_warn} Unerwartetes Argument `{arg}`. Erwartet `{arg_expected}` bekommen `{arg_provided}`\n{emote_wrench} **Korrekte Benutzung:** {usage}",
+	"event_message_args_missing": "{emote_warn} Fehlendes Argument `{arg}`. Erwartet `{arg_expected}` bekommen `{arg_provided}`\n{emote_wrench} **Korrekte Benutzung:** {usage}",
+	"event_message_args_unexpected_list": "{emote_warn} Kann nicht analysieren `{arg}` als `{arg_expected}`.\n{emote_wrench} **Richtige Benutzung:** {usage}",
+	"event_message_args_missing_list": "{emote_warn} Fehlende argumente `{arg}`. Erwartet `{arg_expected}` bekommen `{arg_provided}`\n{emote_wrench} **Richtige Benutzung:** {usage}",
+	"event_message_edit": "{emote_edit} Nachricht von **{target_tag}** `{target_id}` wurde Bearbeitet in <#{channel_id}>\n`ID (channel-message): {after_channel_id}-{after_id}`\n**B:** {before}\n**A:** {after}",
+	"event_message_delete": "{emote_trash} Nachricht von **{target_tag}** `{target_id}` wurde gelöscht in <#{channel_id}>\n`ID (channel-message): {after_channel_id}-${after_id}`\n**C:** {content} \n**A:** {attachment}\n**E:** {embed}",
```

Please translate or give context to the "C A E" used in the latter part of this string.
```suggestion
	"event_message_delete": "{emote_trash} Nachricht von **{target_tag}** `{target_id}` wurde in <#{channel_id}> gelöscht\n`ID (Kanal-Nachricht): {after_channel_id}-${after_id}`\n**Inhalt:** {content} \n**Angehängte Datei:** {attachment}\n**Embed:** {embed}",
```